queries.sql

```diff
@@ -29,3 +30,111 @@ WHERE name <> 'Gabumon';
 -- Query 8: Find all animals with a weight between 10.4kg and 17.3kg (including animals with weights equal to 10.4kg or 17.3kg)
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+*/
+
+BEGIN; -- Begin the FIRST transaction
+
+BEGIN; -- Begin the transaction
+
+-- Update the animals table by setting the species column to "unspecified" for all animals
+UPDATE animals
+SET species = 'unspecified';
+
+-- Verify the changes
+SELECT * FROM animals;
+
+ROLLBACK; -- Roll back the transaction
+
+-- Verify that the changes were rolled back
+SELECT * FROM animals;
+
+
+BEGIN; -- Begin the SECOND transaction
+
+-- Update the animals table by setting the species column to "digimon" for animals with names ending in "mon"
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+-- Update the animals table by setting the species column to "pokemon" for animals without a species already set
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
```

Comment:
- [ ] Because you inserted data into the species column initially, this command that sets the species column that is `null` to Pokemon did not work as expected because those columns had values, hence are not `null`.  The solution to this issue is - _Do not add any data into the species column in the first place._ 


<img width="498" alt="image" src="https://github.com/redmarverick/Animals-DB/assets/84156714/d3ebec8e-9645-476c-9082-ec74e23b4341">